x-pack/filebeat/input/httpjson/request.go

```diff
@@ -207,7 +311,99 @@ func (r *requester) doRequest(stdCtx context.Context, trCtx *transformContext, p
 		n++
 	}
 
+	httpResp.Body.Close()
 	r.log.Infof("request finished: %d events published", n)
 
 	return nil
 }
+
+// getJSON returns array of string values from json string
+func getJSON(b, str string) ([]string, error) {
```

Comment:
Could you please extract it to a helper or any other place?

Reply:
Yeah, Right.
I will do it. Thanks!